server/schema.sql

```diff
@@ -0,0 +1,60 @@
+DROP DATABASE IF EXISTS reviewsservice;
+CREATE DATABASE reviewsservice;
+
+\c reviewsservice;
+
+CREATE TABLE
+  reviews(
+    id SERIAL PRIMARY KEY NOT NULL,
+    product_id SERIAL NOT NULL,
+    rating smallint,
+    date bigint NOT NULL,
+    summary TEXT,
+    body TEXT,
+    recommend boolean,
+    reported boolean,
+    reviewer_name VARCHAR(255),
+    reviewer_email VARCHAR(255),
+    response TEXT,
+    helpfulness INTEGER
+);
+
+\copy reviews FROM 'server/data/reviews.csv' DELIMITER ',' CSV HEADER;
+
+
+
+CREATE TABLE
+  review_photos(
+    id SERIAL PRIMARY KEY NOT NULL,
```

Comment:
More preference than anything, but if you are using `SERIAL` you can likely drop the `NOT NULL` condition from the schema.

According to [Postgres Docs](https://www.postgresql.org/docs/14/datatype-numeric.html#DATATYPE-SERIAL) it appears that `SERIAL` has that condition included by default